trillian/integration/ct_integration_test.cfg

```diff
@@ -1,26 +1,39 @@
-[
-	{
-		"LogID": @TREE_ID@,
-		"Prefix": "athos",
-		"RootsPEMFile": ["@TESTDATA@/fake-ca.cert"],
-		"PubKeyPEMFile": "@TESTDATA@/ct-http-server.pubkey.pem",
-		"PrivKeyPEMFile": "@TESTDATA@/ct-http-server.privkey.pem",
-		"PrivKeyPassword": "dirk"
+{
+	"log_id": @TREE_ID@,
+	"prefix": "athos",
+	"roots_pem_file": ["@TESTDATA@/fake-ca.cert"],
+	"public_key": {
+		"der": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEB/hRr6qMVoOQMbeA49Ya9y82BnHs3Tu+fjZvDRwcYAt/9Z//5SRJNFbySxBfvwgf+Q7PNbWKioswClS3vx1NuQ=="
```

Comment:
Could you put some explicit commands in the commit message so it's really easy for someone to reconstruct the data in future.